minio/credentials/providers.py

```diff
@@ -560,15 +560,25 @@ def __init__(
             ldap_username: str,
             ldap_password: str,
             http_client: Optional[PoolManager] = None,
+            *,
```

Comment:
Move `*`, after `self`

Reply:
The `master`  broke many things already. The next release must be `v8.0.0`

Reply:
@ramondeklein I discussed with @balamurugana, he mentioned that, the current `master` is already having breaking changes.
So next release should be a major release: `v8.0.0`